backend/compact-connect/lambdas/python/compact-configuration/handlers/compact_configuration.py

```diff
@@ -219,6 +224,72 @@ def _put_compact_configuration(event: dict, context: LambdaContext):  # noqa: AR
         raise CCInvalidRequestException('Invalid compact configuration: ' + str(e)) from e
 
 
+def _validate_configured_states_transitions(
+    existing_states: list[dict], new_states: list[dict], compact: str, submitting_user_id: str
+) -> None:
+    """
+    Validate that configuredStates transitions are allowed.
+
+    Rules:
+    1. States cannot be manually added or removed - only managed internally by the API
+    2. Only isLive status can be modified (false -> true only)
+
+    :param existing_states: Current configuredStates from the database
+    :param new_states: New configuredStates from the request
+    :param compact: The compact abbreviation for logging
+    :param submitting_user_id: The user making the request for logging
+    :raises CCInvalidRequestException: If validation fails
+    """
+
+    # Create lookup dictionaries for easier comparison
+    existing_states_by_postal = {state['postalAbbreviation']: state for state in existing_states}
+    new_states_by_postal = {state['postalAbbreviation']: state for state in new_states}
+
+    # Check for removed states
+    removed_states = set(existing_states_by_postal.keys()) - set(new_states_by_postal.keys())
+    if removed_states:
+        logger.warning(
+            'Attempt to remove states from configuredStates',
+            compact=compact,
+            submitting_user_id=submitting_user_id,
+            removed_states=list(removed_states),
+        )
+        raise CCInvalidRequestException(
+            f'States cannot be removed from configuredStates. Attempted to remove: {", ".join(sorted(removed_states))}'
+        )
+
+    # Check for added states
+    added_states = set(new_states_by_postal.keys()) - set(existing_states_by_postal.keys())
+    if added_states:
+        logger.warning(
+            'Attempt to add states to configuredStates',
+            compact=compact,
+            submitting_user_id=submitting_user_id,
+            added_states=list(added_states),
+        )
+        raise CCInvalidRequestException(
+            f'States cannot be manually added to configuredStates. Attempted to add: {", ".join(sorted(added_states))}'
+        )
+
+    # Check for isLive downgrades (true -> false)
+    for postal_abbr, existing_state in existing_states_by_postal.items():
+        if postal_abbr in new_states_by_postal:
+            new_state = new_states_by_postal[postal_abbr]
+            if existing_state['isLive'] is True and new_state['isLive'] is False:
```

Comment:
Similarly, can we just do 
```suggestion
            if existing_state['isLive'] and not new_state['isLive']:
```